### PR TITLE
feature: add options and endedAt

### PR DESCRIPTION
**Pull request by YohaLin**
1. add options and endedAt from GET_APP in hasura

### CODE REVIEW

File: src/types/app.ts [hunk 1]
```diff
@@ -115,6 +115,11 @@ export type Currency = {
   minorUnits: number
 }
 
+export type Options = {
```
Comment (Ponchimeow) [inline src/types/app.ts:118]: `Options` is ambiguous
maybe can named `AppOptions`

File: src/types/app.ts [hunk 2]
```diff
@@ -152,6 +157,8 @@ export type AppProps = {
   currencyId: string
   currencies: { [currencyId: string]: Currency }
   navs: AppNavProps[]
+  options: Options
+  endedAt: string
```
Comment (Ponchimeow) [inline src/types/app.ts:161]: endedAt is timestamp with time zone or nullable in table
type should be Date | null

**Review by Ponchimeow — CHANGES_REQUESTED** [review]

File: src/types/app.ts (hunk 1 above)
Comment (YohaLin) [reply src/types/app.ts:118]: Okay! Thanks

File: src/types/app.ts (hunk 2 above)
Comment (YohaLin) [reply src/types/app.ts:161]: Okay! Thanks

**Review by Ponchimeow — APPROVED** [review]

**Pull request merged**